docs/tutorials/t2ramsey_characterization.ipynb

```diff
@@ -13,9 +13,9 @@
    "source": [
     "This experiment serves as one of the series of experiments used to characterize a single qubit. Its purpose is to determine two of the qubit's properties: *Ramsey* or *detuning frequency* and $T_2\\ast$. The rough frequency of the qubit was already determined previously. The control pulses are based on this frequency. \n",
     "\n",
-    "In this experiment, we would like to get a more precise estimate of the qubit's frequency. The difference between the frequency used for the control rotation pulses, and the precise frequency is called the *detuning frequency*. This part of the experiment is called a *Ramsey Experiment*. $T_2\\ast$ represents the rate of decay toward a mixed state, when the qubit is initialized to the |+⟩ state.\n",
+    "In this experiment, we would like to get a more precise estimate of the qubit's frequency. The difference between the frequency used for the control rotation pulses, and the precise frequency is called the *detuning frequency*. This part of the experiment is called a *Ramsey Experiment*. $T_2\\ast$ represents the rate of decay toward a mixed state, when the qubit is initialized to the $\\left|1\\right\\rangle$ state.\n",
```

Comment:
$T_2\ast$ is usually styled as $T_2^\ast$, should we adopt that?

Reply:
yes, superscript on the * is nicer. Done